ci/blackbox/integration_suite_test.go

```
@@ -45,7 +45,7 @@ func TestSuite(t *testing.T) {
 
 		fmt.Fprintln(GinkgoWriter, os.Environ())
 
-		originalConfig, err := broker.LoadConfig("./config.json")
+		originalConfig, _ := broker.LoadConfig("./config.json")
```

Deliberately discarding any errors here rather than checking them?